doc/manual/src/plugins.md

```diff
@@ -0,0 +1,277 @@
+# Plugins
+
+This chapter describes all plugins present in hydra
+
+### Inputs
+
+Hydra supports the following inputs
+
+- Bazaar input
+- Darcs input
+- Git input
+- Mercurial input
+- Path input
+
+## Bitbucket pull requests
+
+Create jobs based on open bitbucket pull requests
+
+configuration options:
+- `bitbucket_authorization.<owner>`
```

Comment:
In general, what do you think about:

* making the first line a full sentence and adding a period at the end?
* making `configuration options:` a header?

Like this:

```suggestion
## Bitbucket pull requests

Create jobs based on open Bitbucket pull requests.

### Configuration Options
- `bitbucket_authorization.<owner>`
```